charts/portworx/values.yaml

```diff
@@ -108,7 +108,7 @@ aws:
   accessKeyId: 
   secretAccessKey: 
 
-deleteType:
+deleteType: UninstallAndWipe
```

Comment:
Will this install PX or not?

Reply:
This field is different from the deletestrategy field. This value is used for the shell script that patches the STC before deletion and doesn't interfere with installation